tests/scripts/test_snactor.py

```diff
@@ -30,6 +31,16 @@ def test_discovery(repository_dir):
             check_call(['snactor', 'discover'])
 
 
+def test_raises_create_outside_repository():
+    for cmd in ([['snactor', 'new-tag', 'winteriscoming'],
+                 ['snactor', 'new-topic', 'winteriscoming'],
+                 ['snactor', 'new-model', 'winteriscoming', '--topic', 'WinteriscomingTopic']]):
+        with pytest.raises(CalledProcessError) as err:
+            check_output(cmd, stderr=STDOUT)
+        assert ('This command must be executed from the repository directory' in
+                err.value.stdout.decode('utf-8'))
```

Comment:
From the CI:
```AttributeError: 'CalledProcessError' object has no attribute 'stdout'```

This is due to python 3 introducing those fields and the code must be Python 2 compatible.

Reply:
my bad, should be fixed by using value.ouput